src/languageservice/serviceclient.ts

```diff
@@ -177,6 +181,7 @@ export default class SqlToolsServiceClient {
                 }).catch(err => {
                     Utils.logDebug(Constants.serviceLoadingFailed + ' ' + err );
                     Utils.showErrorMsg(Constants.serviceLoadingFailed);
+                    Telemetry.sendTelemetryEvent('ServiceInitializingFailed');
```

Comment:
Exception telemetry in already send in mainController for all type of exception happens while initializing, I thought we may want to also have the number of times the service initializing fails 

---
In reply to: [91185018](https://github.com/Microsoft/vscode-mssql/pull/462#discussion_r91185018) [](ancestors = 91185018)